Bindings/Java/Matlab/tests/testOsimVec3.m

```diff
@@ -0,0 +1,28 @@
+%% clear working space
+clear all;close all;clc;
+%% import opensim libraries
+import org.opensim.modeling.*
+% Adding a call to Model(). If Model() is not instantiated, the test fails.
+m = Model();
```

Comment:
I have to instantiate a model or the ctest will fail. This shouldn't be needed for the test, however, if this is not included the test will fail. 

```
107: An exception has been thrown during the execution
107:   JavaException with properties:
107: 
107:     ExceptionObject: [1x1 java.lang.UnsatisfiedLinkError]
107:          identifier: 'MATLAB:Java:GenericException'
107:             message: 'Java exception occurred: ...'
107:               cause: {}
107:               stack: [1x1 struct]
107: 
107:     file: '/Users/jamesdunne/opensim/opensim-core/Bindings/Java/Matlab/tes...'
107:     name: 'testOsimVec3'
107:     line: 8
107: 
107: 
107: CMake Error at /Users/jamesdunne/opensim/opensim-core/cmake/MatlabTestsRedirect.cmake:112 (message):
107:   [MATLAB] TEST FAILED
```

Reply:
Weird. Could you try running the test as a matlab script without CTest? That might provide more info.

Reply:
+1 for adding the test.

Reply:
I ran as a script and it worked both ways (line 6 commented out or not). In ctests, I am still getting a 'test failed' message if line 6 is commented out. That error message is not really telling.